libs/langchain-community/src/vectorstores/hanavector.ts

```diff
@@ -58,7 +122,7 @@ export class HanaDB extends VectorStore {
 
   private vectorColumnLength: number;
 
-  declare FilterType: Filter;
```

Comment:
You'll still want this for methods on the superclass

Reply:
Thank you for pointing it out! I added this declaration and used it in methods.